exist-core/src/test/xquery/xquery3/serialize.xql

```diff
@@ -254,7 +254,7 @@ function ser:serialize-empty-sequence() {
 
 declare
     %test:args(1234)
-    %test:assertEquals('1234')
+    %test:assertEquals(1234)
```

Comment:
Seems to be unrelated to tumbling window

Reply:
It is simply reseting something that has been incorrectly changed since I forked the code to work on Window Expressions.

Reply:
@adamretter Patrick raises a valid concern here that the changes to serialize.xql in https://github.com/eXist-db/exist/commit/ccfb1d5a38fc4a882968f68c7165744ac1f088eb have no relation the present PR. Please remove this commit from the PR. If the change is needed, it should be addressed in a separate PR.

Reply:
@joewiz Those changes were previously part of a larger set of serialization fixes that were needed for this PR to correctly pass the XQTS. Most of those serialization fixes I extracted and sent in a separate PR that was previously merged. These appear here as an artifact of rebasing on those previously merged changes.

For the sake of getting this merged, I have now removed that commit as you and Patrick requested.